packages/migrate/migrations/routes/utils.js

```diff
@@ -46,7 +47,7 @@ export function error(description, comment_id) {
  */
 export function move_file(file, renamed, content, use_git) {
 	if (use_git) {
-		execSync(`git mv ${JSON.stringify(file)} ${JSON.stringify(renamed)}`);
+		execSync(`git mv ${sq.quote([file, renamed])}`);
```

Comment:
Could we use https://nodejs.org/dist/latest/docs/api/child_process.html#child_processexecfilefile-args-options-callback instead so we can pass in an array of arguments directly and not have to worry about the shell escaping?

Reply:
Untested, but I think it'd be `execFileSync('git', ['mv', file, renamed])`.

Reply:
ohh, thats nice.
Seems to work alright for `\n and $`
execFileSync:
```
node -e 'console.log(require("child_process").execFileSync("echo",["bla\\n\$1"]).toString())'
bla\n$1
```

execSync:
```
node -e 'console.log(require("child_process").execSync("echo bla\\n\$1").toString())'
blan
```

Reply:
ooh yeah that's way better